x/oracle/keeper/reward.go

```diff
@@ -18,7 +18,7 @@ func (k Keeper) RewardBallotWinners(
 	ballotWinners map[string]types.Claim,
 ) {
 	rewardDenoms := make([]string, len(voteTargets)+1)
-	rewardDenoms[0] = types.MicroUmeeDenom
+	rewardDenoms[0] = types.UmeeDenom
```

Comment:
this actually need to be the base denom, i.e. `uumee`.

Just a quick note/recap on denominations in the state-machine:

- The state-machine only deals with and knows about "base-level" denominations, e.g. `uumee`, `uatom`, `uosmo`, etc...In other words, these "base-level" denoms are not divisible, they cannot be broken down any further.
- Denomination metadata exists in `x/bank` and such metadata is used to let users and clients know how to deal with and display "higher-level" denominations, e.g. `1 umee = 10^6 uumee (1000000uumee)`.
- The `x/oracle` module gets prices for high-level denominations, e.g. `atom` and thus we will use the metadata to do the conversions.